test/mlir/conversion/onnx_to_tosa/Tensor/Reshape.mlir

```diff
@@ -7,6 +7,19 @@ func.func @test_static_reshape(%arg0: tensor<32x512x1x1xf32>) -> tensor<32x512xf
   //CHECK:  %[[VAR0:.*]] = "tosa.reshape"(%arg0) {new_shape = [32, 512]} : (tensor<32x512x1x1xf32>) -> tensor<32x512xf32>
 }
 
+// -----
+
+func.func @test_static_reshape_zero(%arg0: tensor<32x512x1x1xf32>) -> tensor<32x512xf32> {
+  %0 = "onnx.Constant"() {value = dense<[0,-1]> : tensor<2xi64>} : () -> tensor<2xi64>
+  %1 = "onnx.Reshape"(%arg0, %0) : (tensor<32x512x1x1xf32>, tensor<2xi64>) -> tensor<32x512xf32>
+  return %1 : tensor<32x512xf32>
+// CHECK-LABEL:   func.func @test_static_reshape_zero(
+// CHECK-SAME:                                        %[[VAL_0:.*]]: tensor<32x512x1x1xf32>) -> tensor<32x512xf32> {
+// CHECK:           %[[VAL_1:.*]] = "tosa.const"() {value = dense<[0, -1]> : tensor<2xi64>} : () -> tensor<2xi64>
```

Comment:
This check doesn't seem relevant for the reshape rewrite - please remove if it isn't.